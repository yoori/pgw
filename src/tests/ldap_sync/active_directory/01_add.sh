#!/bin/sh
# OUT:Modifyfred

if [ -z $ACTIVE_DIRECTORY_TEST_SERVER ]; then
  echo ACTIVE_DIRECTORY_TEST_SERVER not defined
  exit 1;
fi

#
#  If using a "remote" server, ssh key auth must be set up so
#  samba-tool can be run as root to perform modifications.
#

CMDSTART="sudo "
if [ "$ACTIVE_DIRECTORY_TEST_SERVER" != "127.0.0.1" ]; then
CMDSTART="ssh root@$ACTIVE_DIRECTORY_TEST_SERVER "
fi

#
#  Create a group, computer and user.  Samba will notify about all
#  three, the filter in the sync config means only the user will be processed.
#

${CMDSTART}samba-tool group add dummy > /dev/null
${CMDSTART}samba-tool computer create test_workstation > /dev/null
${CMDSTART}samba-tool user create fred asdf_1234 > /dev/null
