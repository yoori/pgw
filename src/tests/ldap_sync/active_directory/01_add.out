Modify CN=Users,dc=example,dc=com CN=fred,CN=Users,DC=example,DC=com fred 512 
